hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -117,6 +118,9 @@ set_ignore_cdc_paths -name async_fifo_to_ibex_ecc_err -through_signal $async_fif
 set_ignore_cdc_paths -name async_fifo_to_ibex_ivalid -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/instr_rvalid_i]
 set_ignore_cdc_paths -name async_fifo_to_ibex_dvalid -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/data_rvalid_i]
 
+# CDC between tlul_fifo_async and regs : ignored
+set tlul_async_data [get_pins -of_objects [get_cells -hier * -filter {ref_name == tlul_fifo_async}] -filter {name =~ tl_d_o*}]
+set_ignore_cdc_paths -name tlul_async_fifo_err -through_signal $tlul_async_data -through_signal [get_pins {top_earlgrey/*/u_reg/u_io_*meas_ctrl_shadowed_cdc/src_we_i}]
```

Comment:
Hm... is this really a common synchronizer primitive?

`*meas_ctrl_shadowed_cdc` only seems to be present inside the clock manager...
should we move this waiver to the associated block leve waiver instead?

Reply:
this particular one might become more common.. it's technically applicable to any block that uses the async reg construct.
Do you not see more like this in your setup @jeoongp ?